lib/dockerregistry/storage_driver.go

```diff
@@ -54,8 +56,15 @@ import (
 // Name of storage driver.
 const Name = "kraken"
 
-func init() {
```

Comment:
Can you file an issue to remove the inits across all the packages under github issue for this project ?